contents/handbook/growth/sales/how-to-do-discovery

```diff
@@ -0,0 +1,264 @@
+
+This is meant to be a guide, not a rule-set. Individuals at PostHog are just that, individuals. Each person has their own unique style, delivery, tone, cadence. The goal here is to surface the right insights, by providing a framework for how to go about _asking the right questions_ vs. a talk-track for how to run discussions with prospects and customers. 
+  
+**Core Principles:**
+
+- **Curiosity over pitching** - Be genuinely interested in their challenges
+- **Dig for the 'why'** - Every answer leads to another question
+- **Find the pain** - People buy solutions to problems, not features
+- **Identify champions** - Someone needs to sell internally when you're not there
+- **Build relationships** - Trust is earned through understanding, not just talking
+
+## Why Discovery Matters at PostHog
+PostHog is a broad product suite with many nuances depending on the use case. Discovery helps us provide customers with a better experience by understanding their specific needs so we can:
+
+- **Conduct a demo that actually matters** - No one wants to sit through features they'll never use
+- **Draw connections between their problems and our products** - There are 10+ products (and counting), we want to help them find the right combination
+- **Skip the irrelevant stuff and get to the good bits** - Customers' time is valuable, and generic sales calls aren't
+- **Reduce time & effort needed for them to make a confident decision** - By understanding requirements upfront, we can address concerns early and focus on what matters most to stakeholders
+
+This isn't about qualifying customers through a rigid framework - it's about delivering value through understanding what they actually care about. 
+
+## Timeline for Discovery
+
+Depending on the context, there is normally no pressure to get all of your discovery questions answered on the first call. Discovery happens over the course of your engagement with the customer/prospect and isn't a 'box' you can check off of your to-do list. Beyond your first interaction with a customer/prospect, there will be other opportunities + mediums for which you can do deeper discovery and build a relationship over a longer time-horizon. 
+
+Sometimes this can work to your benefit as, like with most relationships, they are not often built overnight. 
+
+### Other channels
+
+Beyond the 1st call, there are other channels which you can use for discovery:
+
+**Other channels you can use for discovery:** 
+- Zoom / Phone
+- Email
+- Shared team Slack channel
+- DMs in Slack / LinkedIn / X
+- Text / SMS
+
+Leverage these wisely by using your best judgement in combination with asking yourself the question "What would PostHog do?"
+
+## Before your 1st call
+
+### Prep work
```

Comment:
Not for this handbook page but I'd love for us to think about how this can be more automated/easily surfaced to team members so that less digging around in these tools is required.